clms-api/src/main/java/com/clms/api/courses/api/package-info.java

```diff
@@ -0,0 +1,6 @@
+@ApplicationModule(
+        type = ApplicationModule.Type.OPEN
+)
+package com.clms.api.courses.api;
+
+import org.springframework.modulith.ApplicationModule;
```

Comment:
this allows the course/api folder to be exposed to the rest of the application.  In a multi-repo application this could be a java dependency that multiple other repos bring in to use the API